gql/lib/src/language/parser.dart

```diff
@@ -315,13 +315,13 @@ class _Parser {
 
   ValueNode _parseNonConstValue() => _parseValue(isConst: false);
 
-  ValueNode _parseValue({bool isConst}) {
-    final token = _next();
+  ValueNode _parseValue({bool? isConst}) {
+    final token = _next()!;
```

Comment:
@klavs applied [the previous changes to `parser.dart`](https://github.com/gql-dart/gql/pull/181/files#diff-8eae12e33d2d5ab4ad734534b186e82d6f7e0a43684712daa2a6cd3a6c4b38f1), but a lot of these I don't really understand the expected behavior of yet